sklearn/datasets/descr/covtype.rst

```diff
@@ -8,10 +8,19 @@ collected for the task of predicting each patch's cover type,
 i.e. the dominant species of tree.
 There are seven covertypes, making this a multiclass classification problem.
 Each sample has 54 features, described on the
-`dataset's homepage <http://archive.ics.uci.edu/ml/datasets/Covertype>`__.
+`dataset's homepage <http://archive.ics.uci.edu/ml/datasets/Covertype>`_.
```

Comment:
Why did you change this? "__" means anonymous link, and it was probably a good choice here.

Reply:
probably bad copy paste... reverted